contrib/packs/tests/test_action_aliases.py

```diff
@@ -0,0 +1,106 @@
+from st2tests.base import BaseActionAliasTestCase
+
+
+class DeployActionAliasTestCase(BaseActionAliasTestCase):
+    action_alias_name = 'pack_deploy'
+
+    def test_pack_deploy_alias(self):
+        # Includes packs
+        format_string = self.action_alias_db.formats[0]['representation'][0]
+
+        # Command with branch
+        command = 'pack deploy StackStorm/st2contrib packs libcloud,aws branch master'
+        expected_parameters = {
+            'repo_name': 'StackStorm/st2contrib',
+            'packs': 'libcloud,aws',
+            'branch': 'master'
+        }
+        self.assertExtractedParametersMatch(format_string=format_string,
+                                            command=command,
+                                            values=expected_parameters)
```

Comment:
Renamed from `values` to `parameters` in f0fb10168b4b4e7b54946fb3804d9cef5e18141b.